newIDE/app/src/InAppTutorial/SchemaChecker.js

```diff
@@ -30,6 +30,8 @@ const flowStepKeys = [
   'tooltip',
   'skippable',
   'isOnClosableDialog',
+  'interactsWithCanvas',
```

Comment:
new prop to "unlock" canvas if a user is supposed to interact with it (placing objects)

Reply:
To add on the documentation

Reply:
done